jsonnet/components/prometheus-adapter.libsonnet

```
@@ -39,7 +39,9 @@ function(params)
       },
     },
 
-    clusterRoleAggregatedMetricsReader+:
+    // This role is moved to cluster-monitoring-operator.libsonnet
+    // Hence hidding it in prometheus-adapter
+    clusterRoleAggregatedMetricsReader::
```

(nit) better to set to an empty map or null.
```suggestion
    clusterRoleAggregatedMetricsReader:: {},
```